gradle/libs.versions.toml

```diff
@@ -135,6 +136,7 @@ jsqlparser = { group = "com.github.jsqlparser", name = "jsqlparser", version.ref
 mysql-driver = { group = "mysql", name = "mysql-connector-java", version.ref = "mysql" }
 postgresql-driver = { group = "org.postgresql", name = "postgresql", version.ref = "postgresql" }
 commons-cli = { group = "commons-cli", name = "commons-cli", version.ref = "commons-cli" }
+javax-activation = { group = "javax.activation", name = "activation", version.ref = "javax-activation" }
```

Comment:
Can you please update the license bin file accordingly?

Reply:
You don't fix this.

Reply:
Done, I'm trying to find the root cause of this issue.